docs/dgx-pod.md

```diff
@@ -309,14 +350,70 @@ It is also possible to remove the DGX from kubernetes and reserve the resources
 
 Once the DGX compute nodes have been added to Kubernetes and Slurm, you can use the `scripts/doctl.sh` script to manage which scheduler each DGX is allowed to run jobs from.
 
-### 8. Deploy additional services
 
-Deploy additional (optional) components on top of Kubernetes by following the [Optional Components](kubernetes-cluster.md#optional-components) section of the Kubernetes Cluster Guide.
+### 8. Deploy additional cluster services
+
+The following components are completely optional and can be installed on an existing Kubernetes cluster.  They are recommended.
+
+
+### Kubernetes Dashboard
```

Comment:
Each of these headers should have 4 `#` (to be under the main one, which is 3 `#`).